code/game/machinery/doors/windowdoor.dm

```diff
@@ -435,8 +436,8 @@
 	base_state = "right"
 
 /obj/machinery/door/window/antitheft
-	desc = "Better do not try to steal, pal, lube off."
 	name = "Branch, do not add stuff here"
+	desc = "Better do not try to steal, pal, lube off."
```

Comment:
```suggestion
	desc = "Better not try to steal, pal, lube off."
```

Reply:
I was just moving this comment, and I think it is fine as it is.